src/main/kotlin/io/inventi/eventstore/eventhandler/IdempotentEventHandler.kt

```diff
@@ -48,21 +54,28 @@ abstract class IdempotentEventHandler(
     @Autowired
     private lateinit var transactionTemplate: TransactionTemplate
 
-
     var running: Boolean = false
 
     override fun start() {
         ensureSubscription()
 
         val persistentSubscription = object : PersistentSubscriptionListener {
 
-            override fun onClose(subscription: PersistentSubscription?, reason: SubscriptionDropReason?, exception: Exception) {
-                if (exception is EventStoreException) {
-                    logger.warn("Eventstore connection lost: ${exception.message}")
-                    logger.warn("Reconnecting into StreamName: $streamName, GroupName: $groupName")
-                    eventStore.subscribeToPersistent(streamName, groupName, this)
-                } else {
-                    throw exception
+            override fun onClose(subscription: PersistentSubscription?, reason: SubscriptionDropReason, exception: Exception?) {
+                logger.warn("Subscription was closed. Reason: $reason")
```

Comment:
Would it be possible to log which stream name and group name are we operating on?